raiden/tests/unit/test_startup.py

```diff
@@ -179,7 +201,7 @@ def test_setup_proxies_all_addresses_are_given():
     contracts = {}
     blockchain_service = MockChain(network_id=network_id, node_address=make_address())
 
-    with patched_get_for_succesful_pfs_info():
+    with patch.object(pathfinding, "get_pfs_info", return_value=PFS_CONFIG.info):
```

Comment:
If all you use is the `PFSInfo`, maybe just instantiate this above.

Reply:
I might not understand that, will check